src/bot.py

```diff
@@ -75,40 +123,43 @@ def tweet_loop(self, api):
                 else:
                     api.update_status(self.generate_batch())
                     logging.info("Tweeted out a new flower bed!")
-                    logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
+                    logging.info("The next tweet is scheduled to be made in {} seconds".format(cooldown))

```

Comment:
This is a valid mistake that I'd be happy to address-- in another issue and pull request.

Reply:
done. I will do it

Reply:
> done. I will do it

I'd appreciate it if we could address this after this pull request.